### ISSUE TITLE
SSL

### ISSUE THREAD

**Issue by ghost** [spoulson/Chatmore#103]
Hello Everyone,
Is possible make SSL connection for irc client from web server to irc server ?

Thank you Slava.

**Comment by spoulson** [comment]
Hello,

I've considered this, but I haven't figured a real good reason to.  IRC is inherently insecure.  Like email, anything sent to an IRC server/network is distributed among other servers and it could be in the clear, logged to files in the clear, etc.

---

Shawn Poulson
shawn@ethn.net

**Comment by volga629** [comment]
Hello,
Reason can be very simple. For example NickServ for freenode is identify users, password in clear text. Another example distributed IRC where vip public ip and then it sends traffic based SRV record over the wan which usually  require SSL to minimize chance for attack and unnecessary service discovery on irc server.

Slava.

**Comment by spoulson** [comment]
Sure, SSL can help obscure private messages.  But, my point is it won't
provide solid security on an insecure platform.

Anyway, I'm not opposed to the idea.  Having SSL is often better than not.
 I would like to work on a rewrite of Chatmore to solve some stability
issues I've experienced at the socket level.  However, my time is very
limited, so I haven't been able to work on it.

I'm curious, though.  How are you using Chatmore?  Does it work well for you?

## 

Shawn Poulson
shawn@ethn.net

**Comment by volga629** [comment]
Yes,  I am using internally for support teams interaction. I found Inspireircd with atheme services plugin pack working quite well for bot (wiki information posting).
